ros/lib/models.py

```diff
@@ -132,6 +132,8 @@ class RhAccount(db.Model):
     id = db.Column(db.Integer, primary_key=True)
     account = db.Column(db.Text, nullable=True)
     org_id = db.Column(db.Text)
+    created_at = db.Column(db.DateTime, default=datetime.datetime.utcnow)
+    updated_at = db.Column(db.DateTime, default=datetime.datetime.utcnow)
```

Comment:
The pull  request and Jira says its about just adding the fields. So is there another pull request for the function and trigger to update updated_at?

Reply:
@upadhyeammit - No plan to add a function as iirc at the moment no update call on account table fields is available in ros-backend. Please correct if it exists anywhere.

Reply:
I have few thoughts,
1. If updated_at is not being updated by any of the actions from the account/user then do we need this field? As then its copy of created_at?
2. If the intention is to see if account/user is involved in using the service considering the fact that we receive the update in the form of pcp data or host data, then maybe updated_at should be updated on every new upload of data from same account? however it will be too many of update calls? then maybe we can just see when was the last report uploaded by any of the systems from customer? and that should be fine?

Reply:
@upadhyeammit  - Removed `updated_at` field for now as no update call in ros-backend.